src/Report/Format.php

```diff
@@ -66,7 +66,7 @@ protected function setFormat($format)
         foreach ($results as $uri => $result) {
           $variables = $this->preprocessResult($profile, $target, $result);
 
-          $info['uri'] = $uri;
+          $info['uri'] = str_replace("/", "_", $uri);
```

Comment:
Can you please throw an inline comment into the code that explains the reason for this? Then this step will make more sense.

Reply:
Done, updated with a comment.